src/current/_data/cloud_releases.csv

```diff
@@ -69,3 +69,4 @@ date,sha
 2023-09-29,null
 2023-10-02,null
 2023-10-04,null
+2023-10-12,null
```

Comment:
Hey @mdlinville , central india is live in production, this can be updated and checked in!